content/news/2021-04-01-mixxx-subscription-model.md

```diff
@@ -0,0 +1,24 @@
+title: "Announcing the Mixxx Subscription Service"
+authors: Jan Holthuis
+tags: subscription, cloud
+date: 2021-04-01 12:00:00
+
+We are happy to announce a major change in our licensing model:
+Starting next month, Mixxx will switch to a new, multi-tiered subscription plan.
+
+Using our Mixxx **Core** plan for only $6.99/month (introductory offer, $9.99/month after the first 3 months), you can get your party started.
+It offers advanced DJ functions like a top-notch library and tag editing for organizing your music and supports playback of a single deck.
+
+If you want to enjoy advanced DJ features such as using a second deck and crossfader, recording your set, streaming to shoutcast servers, check out our Mixxx **Creative** plan for only $9.99/month (introductory offer, $14.99/month after the first 3 months).
+
+The Mixxx **Pro** plan ($19.99/month) even goes one step further and lets you spice up your mix by adding EQs, controller support and samplers!
+
+With our additional **FX** and **DVS** plans ($4.99/month each, can be purchased addition to any of the Core/Creative/Pro plans) you can take your DJing to the next level and open even more possibilities by adding support for Effect Chains and timecode vinyl.
+
+This is an amazing opportunity for us to generate shareholder value and reshape the DJ experience for our customers.
```

Comment:
"For real headliners, the **Bottle Service VIP* plan ($499/program launch) will guarantee the appearance of at least three Instagram Influencers at each of your parties"

Reply:
:rofl: